### PR TITLE
feat: add multiple excluded directories to the test coverage script

### PR DESCRIPTION
**Pull request by evan-forbes**
## Overview

this pr enables us to add multiple directories to the exclusion list for measuring test coverage, and then adds the malicious package to that list

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by evan-forbes** [comment]
wait, how can coverage descrease?
Screenshot from 2023-06-29 17-01-43

File: scripts/test_cover.sh [hunk 1]
```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
-PKGS=$(go list ./... | grep -v '/simapp')
```
Comment (cmwaters) [inline scripts/test_cover.sh:4]: Why did you remove simapp? Is it just legacy?

**Review by cmwaters — APPROVED** [review]
I guess it's expected for code coverage percentage to increase because you've removed lines

File: scripts/test_cover.sh (hunk 1 above)
Comment (evan-forbes) [reply scripts/test_cover.sh:4]: yeah, we don't have a simapp directory here

**Review by rootulp — APPROVED** [review]
Tested by creating a go package named malicious, running this snippet of the script and echo'ing the resulting `PKGS`. The output doesn't contain the malicious package so this LGTM.

```shell
$ ./test.sh
github.com/celestiaorg/celestia-app/app github.com/celestiaorg/celestia-app/app/encoding github.com/celestiaorg/celestia-app/app/test github.com/celestiaorg/celestia-app/cmd/celestia-appd github.com/celestiaorg/celestia-app/cmd/celestia-appd/cmd github.com/celestiaorg/celestia-app/pkg/appconsts github.com/celestiaorg/celestia-app/pkg/appconsts/v1 github.com/celestiaorg/celestia-app/pkg/da github.com/celestiaorg/celestia-app/pkg/inclusion github.com/celestiaorg/celestia-app/pkg/namespace github.com/celestiaorg/celestia-app/pkg/proof github.com/celestiaorg/celestia-app/pkg/shares github.com/celestiaorg/celestia-app/pkg/square github.com/celestiaorg/celestia-app/pkg/wrapper github.com/celestiaorg/celestia-app/proto/celestia/da github.com/celestiaorg/celestia-app/test/cmd/txsim github.com/celestiaorg/celestia-app/test/e2e/cmd/e2e github.com/celestiaorg/celestia-app/test/e2e/pkg github.com/celestiaorg/celestia-app/test/tokenfilter github.com/celestiaorg/celestia-app/test/txsim github.com/celestiaorg/celestia-app/test/util github.com/celestiaorg/celestia-app/test/util/blobfactory github.com/celestiaorg/celestia-app/test/util/keeper github.com/celestiaorg/celestia-app/test/util/network github.com/celestiaorg/celestia-app/test/util/testfactory github.com/celestiaorg/celestia-app/test/util/testnode github.com/celestiaorg/celestia-app/x/blob github.com/celestiaorg/celestia-app/x/blob/ante github.com/celestiaorg/celestia-app/x/blob/client/cli github.com/celestiaorg/celestia-app/x/blob/client/testutil github.com/celestiaorg/celestia-app/x/blob/keeper github.com/celestiaorg/celestia-app/x/blob/types github.com/celestiaorg/celestia-app/x/blob/types/test github.com/celestiaorg/celestia-app/x/mint github.com/celestiaorg/celestia-app/x/mint/client/cli github.com/celestiaorg/celestia-app/x/mint/client/testutil github.com/celestiaorg/celestia-app/x/mint/keeper github.com/celestiaorg/celestia-app/x/mint/simulation github.com/celestiaorg/celestia-app/x/mint/test github.com/celestiaorg/celestia-app/x/mint/types github.com/celestiaorg/celestia-app/x/paramfilter github.com/celestiaorg/celestia-app/x/paramfilter/test github.com/celestiaorg/celestia-app/x/qgb github.com/celestiaorg/celestia-app/x/qgb/client github.com/celestiaorg/celestia-app/x/qgb/keeper github.com/celestiaorg/celestia-app/x/qgb/types github.com/celestiaorg/celestia-app/x/tokenfilter github.com/celestiaorg/celestia-app/x/upgrade github.com/celestiaorg/celestia-app/x/upgrade/test
```
